ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+> ℹ Please fill out this template when filing an issue.
```

Comment:
Just an idea to make clear, that this is an informational message. As our repository has also a lot of traffic from beginners, who didn't used GitHub before and might not know about Markdown and how to handle that, I thought it might help to convey that message. Furthermore it could become a helpful marker for automatically processing issues.